src/main/resources/db/migration/V1_1__InsertData.sql

```diff
@@ -10,8 +10,8 @@ VALUES ('c84936e4-2f0f-11e8-b467-0ed5f89f718b','Community Foundation Of Des Moin
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493824-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Planting Bamboo','We are planting lots of Bamboo',now(), null);
 insert into project (proj_guid,org_guid,name,description,start_date,end_date) values ('c8493950-2f0f-11e8-b467-0ed5f89f718b','c849343c-2f0f-11e8-b467-0ed5f89f718b','Big Hug','We are giving lots of Hugs',now(), null);
 
-insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','apanda@gmail.com','Organization Administrator');
-insert into user (user_guid,first_name,last_name,email,role) values ('c84940da-2f0f-11e8-b467-0ed5f89f718b','Mike','Panda','mpanda@gmail.com','Volunteer');
+insert into user (user_guid,first_name,last_name,email,role) values ('c8493c34-2f0f-11e8-b467-0ed5f89f718b','Amanda','Panda','apanda@gmail.com','ROLE_DSMHACK_ADMINISTRATOR');
```

Comment:
versioned flyway migrations are non-repeatable, they really shouldn't be changed. Changing this will make all the migrations fail in any environment where the previous version of this migration was already applied. Note that validate on migrate (https://flywaydb.org/documentation/commandline/migrate#validateOnMigrate) defaults to true

Reply:
I'm learning now that this isn't in production anywhere. perhaps this is a non-issue.